website/docs/r/eventgrid_system_topic_event_subscription.html.markdown

```diff
@@ -0,0 +1,210 @@
+---
+subcategory: "Messaging"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_eventgrid_system_topic_event_subscription"
+description: |-
+  Manages an EventGrid System Topic Event Subscription.
+---
+
+# azurerm_eventgrid_system_topic_event_subscription
+
+Manages an EventGrid System Topic Event Subscription.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "West US 2"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "examplestorageaccount"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+
+  tags = {
+    environment = "staging"
+  }
+}
+
+resource "azurerm_storage_queue" "example" {
+  name                 = "examplestoragequeue"
+  storage_account_name = azurerm_storage_account.example.name
+}
+
+resource "azurerm_system_topic" "example" {
+  name                   = "example-system-topic"
+  location               = "Global"
+  resource_group_name    = azurerm_resource_group.example.name
+  source_arm_resource_id = azurerm_resource_group.example.id
+  topic_type             = "Microsoft.Resources.ResourceGroups"
+}
+
+resource "azurerm_eventgrid_event_subscription" "example" {
```

Comment:
It looks like this example is for `azurerm_eventgrid_event_subscription`. Let's change it over so it's an example of an `azurerm_eventgrid_system_topic_event_subscription`